pkg/kubelet/kubelet.go

```diff
@@ -2238,32 +2239,88 @@ func (kl *Kubelet) rejectPod(pod *api.Pod, reason, message string) {
 		Message: "Pod " + message})
 }
 
+func (kl *Kubelet) getNodeResources() api.ResourceList {
+	nodeResource, err := kl.cadvisor.MachineInfo()
```

Comment:
I saw that the resource information is registered in [initialNodeStatus](https://github.com/kubernetes/kubernetes/blob/93a0eac4768322cb88b515a17f17c10715e896a7/pkg/kubelet/kubelet.go#L969), and that is all I need. So I think just using [initialNodeStatus](https://github.com/kubernetes/kubernetes/blob/93a0eac4768322cb88b515a17f17c10715e896a7/pkg/kubelet/kubelet.go#L969) is enough.